src/pymongo_migrate/mongo_migrate.py

```diff
@@ -32,6 +32,13 @@ def _deserialize(data, cls):
     return cls(**data)
 
 
+def _measure_time(func, argument):
+    start = time.clock() 
```

Comment:
using `time.clock()` is an error here - `time.clock()` counts the CPU time taken by the process - and in case of db migrations, most time will be taken by database itself, so it the python process itself will often release the CPU, thus the time measured will be much lower than wall clock time

Reply:
Right, I was not thinking of that...